Cargo.toml

```diff
@@ -179,11 +180,12 @@ base64-simd         = "0.8"
 cfg-if              = "1.0.0"
 schemars            = "0.8.21"
 oxc-browserslist    = "1.0.1"
+prettyplease        = "0.2.20"
 criterion2          = { version = "0.11.0", default-features = false }
 daachorse           = { version = "1.0.0" }
 
 [workspace.metadata.cargo-shear]
-ignored = ["napi", "oxc_traverse"]
+ignored = ["napi", "oxc_traverse", "oxc_ast_codegen", "prettyplease"]
```

Comment:
Small nit: Does `prettyplease` need to be included here as well as in `tasks/ast_codegen/Cargo.toml`?

Reply:
That's why I told you to only review the up stack PR, It is one of those small changes I did in the next PR and haven't cherry-picked to merge here.

Reply:
Yes, it does, otherwise I get errors from shear.

Reply:
Ah OK.